src/components/Landing/Landing.tsx

```diff
@@ -10,6 +10,7 @@ const Landing = () => {
     { text: "Join Form", link: "/join" },
     { text: "NN Assign Form", link: "/nn-assign" },
     { text: "Query Form", link: "/query" },
+    /*{ text: "Pano", link: "/pano/view/" },*/
```

Comment:
Are you intending to uncomment this in a follow-up PR?

Reply:
Probably not. We actually want to remove this page from Meshforms altogether.